public/app/features/explore/ExploreActions.tsx

```diff
@@ -65,6 +70,24 @@ export const ExploreActions = () => {
         });
       }
     } else {
+      // command palette doesn't know what pane we're in, only show option if not split and no datasource is mixed
+      const hasMixed = Object.entries(panes)
+        .map((pane) => pane[1])
+        .some((pane: ExploreItemState | undefined) => {
+          return pane?.datasourceInstance?.uid === MIXED_DATASOURCE_NAME;
+        });
+
+      if (config.featureToggles.correlations && canWriteCorrelations && !hasMixed) {
+        actionsArr.push({
+          id: 'correlations-editor',
```

Comment:
should probably prefix this id with `explore/` like the others, and also add `section: 'explore'` to the object so it appears in the correct section in the command palette 👍

Reply:
Makes sense! Fixed!